packages/manager/modules/nasha/src/dashboard/partition/accesses/accesses.routing.js

```diff
@@ -0,0 +1,25 @@
+import { createTaskTrackerStateOptions } from '../../../components/task-tracker';
+import { STATE_NAME } from './accesses.constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(STATE_NAME, {
+    url: '/accesses',
+    component: 'nashaDashboardPartitionAccesses',
+    resolve: {
+      breadcrumb: () => null,
+      aclTypeEnum: (schema) =>
```

Comment:
```suggestion
      aclTypeEnum: /* @ngInject */ (schema) =>
```

Reply:
:+1: 
This resolve has been removed : https://github.com/ovh/manager/pull/7494/files#diff-c0f115b196232bd2e7d6bb8a515d38990049029d95bc8261d22a9069a6a7eebfL10-L11
(sorry for that)